### DISCUSSION TITLE
Connecto to API using Hawk Auth

### DISCUSSION THREAD

**Discussion by t3b4n** [airbytehq/airbyte#39354] [Connector Questions]
Hi! Has anyone been able to connect to an API as source using Hawk authentication? If so, can you please guide me on how to implement it? 
I've been able to successfully connect to the API using Python, but Airbyte doesn't seem to have a way to use Hawk auth.

[a comment by an automated account is not shown]

**Comment by jcolinger3** [comment]
I also have a need to know about this.
